python/lsst/summit/utils/tmaUtils.py

```diff
@@ -908,6 +910,10 @@ def __setitem__(self, index: int, value: Any) -> None:
     def __len__(self) -> int:
         return len(self.keys)
 
+    def __iter__(self) -> Any:
+        for key in self.keys:
+            yield self.dictionary[key]
+
```

Comment:
Out of interesting, what was necessitating this?

Reply:
Huh, that feels a _little_ like a mypy bug, but I guess it's probably not. That's fine then, thanks for explaining.